src/Microsoft.ML/CSharpApi.cs

```diff
@@ -15777,9 +15777,8 @@ public SentimentAnalyzerPipelineStep(Output output)
     namespace Transforms
     {
 
-        /// <summary>
-        /// Transforms the data using the TensorFlow model.
-        /// </summary>
+        /// <include file='../Microsoft.ML.TensorFlow/doc.xml' path='doc/members/member[@name="TensorflowTransform"]/*' />
+        /// <include file='../Microsoft.ML.TensorFlow/doc.xml' path='doc/members/example[@name="TensorflowTransform"]/*' />
```

Comment:
>'doc/members/example [](start = 68, length = 20)

I see that `example` tag is under `member` tag. It's not immediately under members. Will it work?

My understanding is that if we are doing
`/// <include file='../Microsoft.ML.TensorFlow/doc.xml' path='doc/members/member[@name="TensorflowTransform"]/*' />`

Everything including example will also be copied here.

Reply:
thanks Zeeshan. 

---
In reply to: [216475085](https://github.com/dotnet/machinelearning/pull/867#discussion_r216475085) [](ancestors = 216475085)